packages/dashboard/tsconfig.json

```diff
@@ -0,0 +1,43 @@
+{
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+        "target": "es2020",
+        "module": "es2020",
+        "jsx": "react",
+        "sourceMap": true,
+        "removeComments": true,
+
+        /* Checks */
+        "strict": true,
+        "noImplicitAny": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "strictBindCallApply": true,
+        "strictPropertyInitialization": true,
+        "noImplicitThis": true,
+        "alwaysStrict": true,
```

Comment:
Setting `strict` as true would cover all these cases, so I think we can omit these checks.

Reply:
Oh cool, let's remove the individual rules. 👍